pkg/node/node.go

```diff
@@ -491,45 +487,6 @@ func NewBee(ctx context.Context, addr string, publicKey *ecdsa.PublicKey, signer
 	if err != nil {
 		return nil, fmt.Errorf("check presence of nonce: %w", err)
 	}
-	if !nonceExists {
```

Comment:
a fresh node may not have a nonce, so this is needed I think

Reply:
overlayNonceExists can be refactored to return an 32 byte zero slice in the case that an old nonce does not exist

Reply:
isn't this already the case?

Reply:
aha yes